yarn-project/sequencer-client/src/test/l2-block-publisher.test.ts

```diff
@@ -14,17 +38,38 @@ const anvilHost = process.env.ANVIL_HOST ?? 'http://127.0.0.1:8545';
 const chainId = 31337;
 
 describe.skip('L1Publisher integration', () => {
```

Comment:
This is currently skipped?

Reply:
The tests were set as skipped before due to requiring an anvil instance running, which was not part of the setup. Maybe @ludamad got insights on better solution. Here simply replacing the old block publisher tests.